Gem/Code/Source/SampleComponentManager.cpp

```diff
@@ -1749,18 +1752,87 @@ namespace AtomSampleViewer
 
         if (xrSystem)
         {
+            RHI::Ptr<RHI::Device> device = Utils::GetRHIDevice();
+            if (RHI::CheckBitsAll(device->GetFeatures().m_shadingRateTypeMask, RHI::ShadingRateTypeFlags::PerRegion) && !m_vrsTexture)
+            {
+                // Need to fill the contents of the Variable shade rating image.
+                const AZStd::shared_ptr<const RPI::PassTemplate> forwardTemplate =
+                    RPI::PassSystemInterface::Get()->GetPassTemplate(Name("MultiViewForwardPassTemplate"));
+                // Find the Shading Rate Attachment
+                Data::Asset<RPI::AttachmentImageAsset> vrsImageAsset;
+                for (const auto& imageAttachment : forwardTemplate->m_imageAttachments)
+                {
+                    if (RHI::CheckBitsAll(imageAttachment.m_imageDescriptor.m_bindFlags, RHI::ImageBindFlags::ShadingRate))
```

Comment:
This looks to be a duplicate of the code in the BootstrapSystemComponent, that seems like a reason the default VRS setup should be moved into Atom proper rather than being passed in from the outside (LyIntegration, or ASV).